ldus/source/include/ldus/list.h

```diff
@@ -2,10 +2,16 @@
 
 #ifndef	LDUS_LIST_H_INCLUDED
 #define LDUS_LIST_H_INCLUDED
-
-#define container_of(ptr, type, member) \
-	((type *)((char *)(ptr) - (size_t)&((type *)0)->member))
-
+
+// get rid of access memory 0x0000000000000000 runtime error which will cause dnet restart.
+#define GET_RID_OF_LIST_ISSUE 1
```

Comment:
Why do you need this constant?

Reply:
Just keep it in case we found other issue of this line of code. If no more issue, then remove it.